### DISCUSSION TITLE
We should have a collection with examples of custom surround objects

### DISCUSSION THREAD

**Discussion by mawkler** [kylechui/nvim-surround#143] [Ideas]
I think we should have a collection of custom surrounds to help and inspire people into configuring surround.nvim. I think this would be especially useful to show examples of how to use it together with treesitter.

**Comment by kylechui** [comment]
These examples should be in #53, but to be honest I've been a bit bogged down by other stuff (internship applications). I'll try to get some more work done on polishing the interface and get `v2.0.0` pushed out before working more on the "user-facing" side of things, since I'm still not 100% happy with how things are at the moment. I also would like to better understand how queries work and if that can be utilized in any meaningful way.
